src/main/resources/translator.properties

```diff
@@ -25,11 +25,10 @@ flow-definition.description = description
 
 com.tikal.jenkins.plugins.multijob.MultiJobProject.description = description
 
-hudson.plugins.buildblocker.BuildBlockerProperty = blockOn
-hudson.plugins.buildblocker.BuildBlockerProperty.type = com.adq.jenkins.xmljobtodsl.dsl.strategies.custom.DSLBlockOnStrategy
```

Comment:
Do we still need the custom type? what does it do?

Reply:
As a custom type, the buildBlockerProperty is currently being skipped over during translation: 
```
description("Test")
	keepDependencies(false)
	properties {
		LeastLoadDisabledProperty {
			leastLoadDisabled(false)
		}
		naginatorOptOutProperty {
			optOut(false)
		}
		heavyJobProperty {
			weight(1)
		}
		throttleConcurrentBuilds()
	}
```

There is a [blockOn](https://jenkins.tinyspeck.com/plugin/job-dsl/api-viewer/index.html#method/javaposse.jobdsl.dsl.jobs.FreeStyleJob.blockOn) property though. Checked these [docs](https://github.com/jenkinsci/job-dsl-plugin/wiki/Migration) and it doesn't seem to be deprecated. Maybe both should be available

Reply:
I just mean, what is the code in the custom type doing? Do we need to try to preserve that or can we delete those files? is it being used for something else?

Reply:
The custom type looks like it's meant to create a similar block under the[ blockOn](https://jenkins.tinyspeck.com/plugin/job-dsl/api-viewer/index.html#method/javaposse.jobdsl.dsl.jobs.FreeStyleJob.blockOn) property. It gets the following properties by their name: useBuildBlocker, blocking jobs, blockLevel, scanQueueFor and formats them as inner methods and parameters in order to return the DSL. This is the only place the custom type is being used so we may be able to delete, but should we still make sure the blockOn property is supported? Or is useBuildBlocker, which is the dynamic dsl, used in place of that ?